lwgenerate/gke/gke.go

```diff
@@ -0,0 +1,259 @@
+package gke
```

Comment:
What is the structure of packages inside lw_generate. Previously the package names related to cloud providers, from this it seems a new package per module?

Reply:
I have the structure of packages matching the structure of CLI commands.  Think command structure is the best level to have this convo.

Reply:
Sorted